### ISSUE TITLE
stopped by <something> string

### ISSUE THREAD

**Issue by liubin** [k0kubun/md2key#15]
If the markdown has something like this, the md2key will exit normally and remain slides will not be generated.

```
# MAINTAINER指令

- MAINTAINER <name>

```

**Comment by liubin** [comment]
Now I found that at least I should escape these chars:

```
        content.gsub!("\"", "\\\"")
        content.gsub!(">", "\\>"")
        content.gsub!("<", "\\<"")
        content.gsub!("$", "\\$") 
```

**Comment by liubin** [comment]
generate xhtml by to_xhtml is :
```
<ul>
<li>MAINTAINER <name></li>
</ul>

```

**Comment by liubin** [comment]
As a workaround , escaped > and <.

```ruby
module Md2key
  class Markdown
    def initialize(path)
      markdown = File.read(path)
      markdown.gsub!("<", "&lt;")
      markdown.gsub!(">", "&gt;")

      xhtml    = to_xhtml(markdown)
```

**Comment by k0kubun** [comment]
Fixed in https://github.com/k0kubun/md2key/commit/a300d5f60c7740f7430d133561862f290f004c3f and released as v0.5.1. :gem:

## input
```md
# MAINTAINER指令

- MAINTAINER <name>
```

## output
![2016-04-23 17 17 13](https://cloud.githubusercontent.com/assets/3138447/14760167/3e2c1ca2-0977-11e6-8aa4-beb3a4c5d7a7.png)
